apps/site/src/lib/styles/zothacks-theme.scss

```diff
@@ -17,3 +17,8 @@ $purple: #6600b6;
 
 // fonts
 $fuzzy-bubbles: "Fuzzy Bubbles", cursive;
+
+// bootstrap overrides
+$font-family-base: $fuzzy-bubbles;
```

Comment:
Issue: `next/font` mangles the actual typeface name, so `Fuzzy Bubbles` cannot be directly used. This was solved in HackAtUCI/HackUCI-Site#23 using a CSS custom property.